src/modules/video_player/index.js

```diff
@@ -94,6 +104,36 @@ class VideoPlayerModule {
             $(VIDEO_PLAYER_SELECTOR).on('click', '.player-overlay.player-fullscreen-overlay', handlePlayerClick);
         }
     }
+
+    hideMouseFullscreen() {
+        const showMouse = () => {
+            $('.video-player--fullscreen')
+                .toggleClass('bttv-hide-cursor-fullscreen', false);
+        };
+
+        const hideMouse = () => {
+            $('.video-player--fullscreen')
+                .toggleClass('bttv-hide-cursor-fullscreen', true);
+        };
+
+        $('body').off('mousemove', '.video-player--fullscreen');
```

Comment:
create a proper function that adds and removes the listener on page load and setting change.. you're creating a listener on body, which does not need to be recreated on every player load..